datafusion/results/single.json

```diff
@@ -1,58 +1,58 @@
 {
     "system": "DataFusion (Parquet, single)",
-    "date": "2023-07-29",
+    "date": "2023-11-19",
     "machine": "c6a.4xlarge, 500gb gp2",
     "cluster_size": 1,
-    "comment": "v28.0.0 (51b4392)",
+    "comment": "v33.0.0 (d2efaa9)",
 
     "tags": ["Rust", "column-oriented", "embedded", "stateless"],
 
     "load_time": 0,
     "data_size": 14779976446,
 
     "result": [
-[2.641, 0.232, 0.216],
-[0.092, 0.074, 0.073],
-[0.159, 0.121, 0.117],
-[0.351, 0.122, 0.120],
-[1.142, 0.874, 0.869],
-[1.399, 1.303, 1.329],
-[0.096, 0.075, 0.076],
-[0.094, 0.074, 0.076],
-[1.516, 1.481, 1.475],
-[2.711, 2.582, 2.583],
-[0.429, 0.332, 0.327],
-[0.558, 0.357, 0.360],
-[1.374, 1.323, 1.336],
-[3.559, 2.670, 2.733],
-[1.519, 1.487, 1.477],
-[1.041, 0.981, 0.988],
-[3.248, 2.748, 2.801],
-[3.139, 2.696, 2.688],
-[7.014, 5.770, 5.775],
-[0.271, 0.111, 0.109],
-[9.975, 1.558, 1.578],
-[11.163, 1.926, 1.881],
-[22.053, 4.208, 4.194],
-[56.007, 12.132, 12.113],
-[2.559, 0.602, 0.586],
-[0.737, 0.479, 0.489],
-[2.548, 0.589, 0.589],
-[9.550, 2.299, 2.286],
-[9.076, 5.399, 5.388],
-[0.579, 0.568, 0.584],
-[2.214, 1.136, 1.136],
-[5.734, 1.593, 1.598],
-[8.357, 7.896, 8.053],
-[11.556, 7.358, 7.369],
-[12.051, 7.878, 7.956],
-[1.866, 1.820, 1.807],
-[0.448, 0.347, 0.358],
-[0.231, 0.197, 0.195],
-[0.248, 0.198, 0.188],
-[0.843, 0.725, 0.722],
-[0.138, 0.091, 0.095],
-[0.115, 0.089, 0.088],
-[0.124, 0.097, 0.092]
+[0.076, 0.045, 0.045],
+[0.110, 0.061, 0.062],
+[0.200, 0.109, 0.106],
+[1.529, 0.117, 0.113],
+[2.265, 0.832, 0.836],
+[2.652, 1.246, 1.247],
+[0.091, 0.065, 0.071],
+[0.100, 0.063, 0.063],
+[2.465, 1.490, 1.478],
+[3.672, 2.539, 2.496],
+[1.627, 0.344, 0.330],
+[1.885, 0.366, 0.369],
+[2.679, 1.253, 1.236],
```

Comment:
Quite some queries regressed, are you sure you are using the same setup as in 
[jychen7](https://github.com/jychen7)
(c6a.4xlarge, 500gb gp2) ?

Reply:
I think it was gp3 storage actually, which is what new instances default to. let me re-run this again and confirm the gp2 500gb storage. I'll put this in draft until I update this and for nickitat's comment.